anthos-bm-gcp-terraform/docs/login.md

```diff
@@ -82,3 +85,137 @@ You can copy this token and use it to login via the Cloud Console.
     (click image to enlarge)
   </em>
 </p>
+
+---
+
+### Login with Google Cloud Identity
+
+Instead of creating a Kubernetes Service Account Token, you can use the email
+identity associated with your Google Cloud Platform account to `Login` to the
+cluster.
+
+<p align="center">
+  <img src="images/gcp_account_login.png" width="600">
+  <em>
+    </br>
+    (click image to enlarge)
+  </em>
+</p>
+
+There are two scenarios for using this method:
+  - [Configure a new cluster for Google Cloud Identity Login](#configure-a-new-cluster-for-google-cloud-identity-login)
+  - [Configure an existing cluster for Google Cloud Identity Login](#configure-an-existing-cluster-for-google-cloud-identity-login)
+
+---
+#### Configure a new cluster for Google Cloud Identity Login
+A new cluster can be configured to allow `Login` using GCP Accounts by
+setting the `spec.clusterSecurity.authorization.clusterAdmin.gcpAccounts`
+field in the cluster configuration. This field accepts a list of GCP Accounts
+that must be allowed to `Login` to the cluster. Upon creation of the cluster
+you should be able to `Login` using any of the provided GCP Accounts. If using
+the Terraform script from this repository then you can simply set the
+[`gcp_login_accounts`](/anthos-bm-gcp-terraform/variables.tf#L181) variable with
+a list of GCP Accounts.
+
+```sh
+apiVersion: baremetal.cluster.gke.io/v1
+kind: Cluster
+metadata:
+  name: cluster1
+  namespace: cluster1-ns
+spec:
+  ...
+  ...
+  ...
+  ...
+  clusterSecurity:
+    authorization:
+      clusterAdmin:
+        gcpAccounts: [foo@gmail.com, bar@gmail.com, bazz@google.com]
+```
+Once the cluster has been created, navigate to the clusters list page in Google
+Cloud Console and select `Login` in the dropdown that appears by clicking the
+verticle-ellipsis icon on the specific cluster's row. From the pop-up that
```

Comment:
nit : is there a screenshot for this section

Reply:
So it's the same screenshots as the earlier Token option. So I didn't want to repeat it across the page.  But now I have added a screenshot of Login -> Selecting the Option in both the top level headings for this page. Let me know if you think it is still unclear